Episode4 azalee updates (#36)
datacarpentry/astronomy-python: _episodes/04-select.md

We call the results `candidate_table` because it contains stars that
are good candidates for GD-1.

Both `sky_point_list` and `pm_point_list` are a set of selection criteria that we
derived from data downloaded from the Gaia Database. To make sure we can repeat

> our analysis at a later date we should save both lists in a file.

Reviewer: ```suggestion
our analysis at a later date we should save both lists to a file.
```

There are several ways we could do that, but since we are already
storing data in an HDF file, let's do the same with these variables.

To save them to an HDF file we first need to put them in a `DataFrame`.

> We've seen how to create a `DataFrame` from an Astropy `Table` and from 

Reviewer: ```suggestion
We have seen how to create a `DataFrame` from an Astropy `Table` and from 
```

ICRS, it is more difficult to identity the stars near the centerline
of GD-1.

We can use our `make_dataframe` function from episode 3 to transform the results back
to the GD-1 frame. In addition to doing the coordinate transformation and reflex correction

> for us this function also compiles everything into a single object (a `DataFrame`) to make it easier to use. Note that because we put this code into a function, we can do all of this with a single line of code!

Reviewer: ```suggestion
for us, this function also compiles everything into a single object (a `DataFrame`) to make it easier to use. Note that because we put this code into a function, we can do all of this with a single line of code!
```
